trunner/test_runner.py

```diff
@@ -20,6 +25,69 @@ def resolve_project_path():
     return project_dir
 
 
+def init_logdir(logdir):
```

Comment:
It would be nice to have type hint for logdir

Reply:
Yeah, I see it now, it will be also corrected in other functions.